src/main/java/com/celements/navigation/Navigation.java

```diff
@@ -77,13 +81,12 @@
 
   private INavigationBuilder navBuilder;
 
-  public static final String LIST_LAYOUT_TYPE = "list";
-
-  private static final String _PAGE_MENU_DATA_TYPE = "pageMenu";
+  @Deprecated
+  public static final String LIST_LAYOUT_TYPE = NavigationConfig.LIST_LAYOUT_TYPE;
 
   public static final String MENU_TYPE_MENUITEM = "menuitem";
 
-  private static Log LOGGER = LogFactory.getFactory().getInstance(Navigation.class);
+  private final static Logger LOGGER = LoggerFactory.getLogger(Navigation.class);
 
   private static final String _NAVIGATION_COUNTER_KEY = NavigationApi.class.getCanonicalName()
       + "_NavigationCounter";
```

Comment:
variables names starting with underscore not allowed according to Google Java Style Guide